jwt/builder.go

```diff
@@ -0,0 +1,128 @@
+/*-
+ * Copyright 2014 Square Inc.
```

Comment:
You can also add your name here if you want since you contributed this entire file (applies to other files too).